### PR TITLE
Fixed Input Issues

### PR DESCRIPTION
**Pull request by tifukelison**
##Input is not able to take empty values

### CODE REVIEW

**Review by FotieMConstant — CHANGES_REQUESTED** [review]
Nice work @Kelison237, However, could you make it display beneath the input field? Not on the warning section? That our be better.

File: assets/js/script.js [hunk 1]
```diff
@@ -40,17 +44,25 @@ createNote = () =>{
...
+     var not = $.trim($('#note').val(''))
     console.log(notes);
 
+
     // store
     localStorage.setItem("notesStore", JSON.stringify(notes));
+   
+} else if (noteText=== ''){
+    console.log('warning, warning, empty note')
+    warning.text('Cannot add empty note 😉');
```
Comment (FotieMConstant) [inline assets/js/script.js]: Can you please make this a separate field? like, make this message to appear in red beneath the text area not in some field as the warning. Thanks

**Review by FotieMConstant — CHANGES_REQUESTED** [review]
Nice work @Kelison237, However, could you make it display beneath the input field? Not on the warning section? That our be better.

**Comment by FotieMConstant** [comment]
Once it's good, please update your PR and it will be merged to main

File: assets/js/script.js (hunk 1 above)
Comment (tifukelison) [reply assets/js/script.js]: Okay

**Review by FotieMConstant — APPROVED** [review]

**Pull request merged**